tests/coverage/coroutine.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(coroutines, coroutine_trait, stmt_expr_attributes)]
+#![feature(coroutines, yield_expr, coroutine_trait, stmt_expr_attributes)]
```

Comment:
The coverage test snapshots will need to be re-blessed:

`./x test coverage --set=build.profiler=true --stage=1 --bless`

(Note that these don't get checked in PR CI, because the particular job we use can't enable the profiler runtime.)